exercism/secret-handshake/secret-handshake.js

```diff
@@ -0,0 +1,44 @@
+//Keeping these comments for future reference:
+//convert decimal to binary using toString method
+//convert binary to decimal using parseInt method
+//check for length of decimal returned
+//answer minus 10, 100, 1000, 10000 depending on length
+//return values for corresponding keys in legend object
```

Comment:
Same thing goes for here. Sometimes you really want to keep comments for reference, just be sure to keep them on a copy of the same file in a different branch, or perhaps in a file thats included by your .gitignore, so that they don't end up in Prrr, where you'll be dinged for it. Alternatively, you could include a tag like "NOTE" at the beginning of the comment, and do a project wide search for "NOTE" and remove the comments right before committing your final changes, push, then add the comments back.